leshan-server-core/src/main/java/org/eclipse/leshan/server/registration/Registration.java

```diff
@@ -262,7 +262,11 @@ public Date getLastUpdate() {
     }
 
     public boolean isAlive() {
-        return lastUpdate.getTime() + lifeTimeInSec * 1000 > System.currentTimeMillis();
+        return isAlive(0);
+    }
+
+    public boolean isAlive(long clemencyInSec) {
```

Comment:
I think this method needs a proper javadoc, in order to understand the meaning of this grace/clemency period.
And the default `isAlive()` as well.

Reply:
I can try